test/parallel/test-module-loading-globalpaths.js

```diff
@@ -29,7 +29,14 @@ if (process.argv[2] === 'child') {
     testExecPath = path.join(prefixBinPath, path.basename(process.execPath));
   }
   const mode = fs.statSync(process.execPath).mode;
-  fs.copyFileSync(process.execPath, testExecPath, COPYFILE_FICLONE);
+  try {
+    fs.copyFileSync(process.execPath, testExecPath, COPYFILE_FICLONE);
+  } catch (e) {
+    if (e.code === 'ENOSPC') {
```

Comment:
IMO the tests shouldn't be changed to account for disk space issues, but rather those issues should be fixed.

Reply:
Yes, we agree, but without knowing when or how to disk the disk space issues we need to do something in the interim to unblock things. The extreme flakiness of the CI has been a real problem and we have to start somewhere. We're not saying "skip!" and that's it, never to be looked at again.

Reply:
A disk space issue is not indicative of a broken behavior in the `node` binary that's being tested. Although this change is made as a reaction to the current state of our CI, it could happen to anyone running our test suite (and IMO no one benefits from having a disk space issue reported as a failure).

Reply:
There are many tests that write to the disk. It makes no sense to change this one because it happened to fail when a CI host had a disk issue.

Reply:
Disk space issue is handled in a build wg issue.

Reply:
It doesn't matter. If the host has no space left, another test will fail after this one was skipped. Or the job will fail after the tests because it cannot write the write the XML result, or something else.

Reply:
so, what do you recommend, removing this commit? @richardlau's comment aligned with skipping this test if space issue occurred.

Reply:
> and IMO no one benefits from having a disk space issue reported as a failure

I's argue that if a machine hasn't enough disk space to complete these tests, the failures would be the least of their concern, and they will experience a *lot* of failures.

I think it's the CI's responsibility to make sure that they can run the test suites responsibility to account for it. (I'm not saying it's any specific teams responsibility, just that this kind of issue should be handled from the CI, not the suite).

Reply:
we need to be aggressive about these kinds of things, or we are going to have less and less contributors.

Reply:
I just removed it for the sake of merging this PR.

Reply:
can someone from the @nodejs/build weigh in on what, if anything, is being done about the persistent infrastructure failures like the the out-of-disk-space issue? These are becoming increasingly blocking as we go.